disperser/batcher/txn_manager.go

```diff
@@ -176,9 +176,15 @@ func (t *txnManager) ensureAnyTransactionEvaled(ctx context.Context, txs []*tran
 	defer queryTicker.Stop()
 	var receipt *types.Receipt
 	var err error
+	// transactions that need to be queried. Some transactions will be removed from this map depending on their status.
+	txnsToQuery := make(map[walletsdk.TxID]*types.Transaction, len(txs))
+	for _, tx := range txs {
```

Comment:
Not this PR, but do these "gas-upping" transactions have different hashes? Why do we need to separately watch for all of them?

Reply:
Each replacing transaction will have a new transaction hash even if it replaces an existing transaction, and not all nodes necessarily are sync'd with the new transaction at the same time. Which means that the old transaction being replaced can be confirmed before the new transaction propagates to the network. Therefore, we'd need to look at all of these transactions to check if any one succeeded.